src/System.Drawing.Common/src/System/Drawing/Graphics.Windows.cs

```diff
@@ -860,29 +860,23 @@ public void DrawBezier(Pen pen, Point pt1, Point pt2, Point pt3, Point pt4)
         /// <summary>
         /// Draws a series of cubic Bezier curves from an array of points.
         /// </summary>
-        public void DrawBeziers(Pen pen, Point[] points)
+        public unsafe void DrawBeziers(Pen pen, Point[] points)
         {
             if (pen == null)
-            {
                 throw new ArgumentNullException(nameof(pen));
-            }
```

Comment:
I'm not sure why these braces are being removed.

Reply:
Not needed per our guidelines, pulls the argument validation in a bit.

Reply:
But also at the cost of making the code bigger in collapsed view 😉

Reply:
> But also at the cost of making the code bigger in collapsed view

I'd call it a plus that you don't have to expand for one line.

Reply:
It really comes down to personal preference 😄 . For example, I find the native code in CoreCLR significantly harder to navigate because of all the one liners that don't have braces.

Reply:
https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md : When editing files, keep new code and changes consistent with the style in the files

The prevalent style in this file is to use extra braces.

Reply:
> The prevalent style in this file is to use extra braces.

I think this is a bit of missing the forest for the trees in this case.

Reply:
What's the forest and what are the trees here?

Reply:
The intent (forest) is making sure files aren't unreadable due to jarring differences in style. The tree is the specifics of the rule. This particular application of the rule is incredibly nitpicky and has no real benefit to readability.

Reply:
I find such changes that aren't necessary distracting from the rest of the PR. And the style guidelines don't say that this new version is preferred, so it's not actually fixing anything, just changing it to your preference; the next person who comes along to edit this file might change it back.

Reply:
> the style guidelines don't say that this new version is preferred

Perhaps they should so we don't rathole on nits...

>  it's not actually fixing anything, just changing it to your preference

The diff isn't hard to follow. I think I'm entitled to cleaning up the methods I'm changing here for style consistency that I prefer and that I personally think makes an improvement in readability.  I've made these methods concise & clean and made a _major_ improvement. I'll end up changing most of the file, so Jeremy's style will be predominant in this one file. _And_ if you look at the other changes I've made outside of this one, you'll see my style is predominant in the other files in this project. Shouldn't there be an argument for dll consistency as well?

> the next person who comes along to edit this file might change it back

If someone else comes in and makes these methods even better than I did (improves them functionally), more power to them- I think they've earned the right to make a minor style tweak in their favor.


Quite frankly this is really disheartening. I've made a big improvement on the way to even more big improvements and the key feedback I've gotten is a long debate over whether I should have removed an unnecessary set of brackets.

All that said, I do understand the intent of the rules and your concerns. I do think about the impact of what I do under those specific criteria (among others) and I try to exercise my best judgement in balancing trade-offs. I think I've chosen well here, or at least well enough that further arguments for alternatives are just niggling.

Reply:
> Quite frankly this is really disheartening. I've made a big improvement on the way to even more big improvements and the key feedback I've gotten is a long debate over whether I should have removed an unnecessary set of brackets.

I'm sorry it's disheartening. That's certainly not the intent. I provided feedback on everything I had to provide feedback on: this, validating the HandleRef usage, and questioning the interop. Your original change (prior to updating it after questions) was only a few lines of product src, so not a ton to comment on, and this style change was a large portion of the diff. You chose to push back on the feedback, from multiple people. When I've received such feedback, I've accepted it and moved on.

Reply:
@JeremyKuhne Feedback similar to what you got here was given number on number of occasions, by multiple different reviewers. We are just being consistent here. For example, here is my comment where I am asking Ashon to submit formatting changes separately from actual code changes: https://github.com/dotnet/coreclr/pull/16223#issuecomment-363641471

It is fine with me if you plan to make significant improvements in these files, want to claim their ownership and change them to your liking. Such style changes should be done in separate PR, and ideally the entire files should be changed to have consistent style, not just a few functions. Here are examples of PRs where component owners are changing the code to their liking  https://github.com/dotnet/corefx/pull/22471 or https://github.com/dotnet/coreclr/pull/17451

Reply:
@jeremykuhne I suggest moving formatting changes in a separate PR. You're right that this code could be better formatted but reviewers are asking not to do it in this change.

Reply:
> We are just being consistent here. For example, here is my comment where I am asking Ashon to submit formatting changes separately from actual code changes

And in that comment you explicitly said to restrict formatting changes to the areas where you're actually changing code. Which I agree with fully and make every attempt to do and I believe I've done in this case. (And I've pushed others to do.)

> It is fine with me if you plan to make significant improvements in these files, want to claim their ownership and change them to your liking. Such style changes should be done in separate PR, and ideally the entire files should be changed to have consistent style, not just a few functions.

I'm happy to do a follow up that makes a pass through the file(s) for consistency.

This caught me off guard. This is the _first_ time I've been tagged for changing the formatting of the code local to what I'm working on (honestly, it is just one line away from "real" changes). It really bothers me as it seems super nitpicky/dogmatic- did you really have a hard time understanding what changed? To me I can't see how it is in the least bit confusing- perhaps I just don't have the right perspective.

I'll put back the brackets. Hopefully my time explaining my perspective isn't wasted. I think we need to be careful about dogma and it worries me that we make things unnecessarily difficult.